src/amazonqGumby/constants.ts

```diff
@@ -0,0 +1,8 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export class CodeTransformConstants {
```

Comment:
Just in general, is there a reason to export constants as classes instead of just exporting the consts or functions separately? This goes for the vast majority of the classes added in this PR, with the exception of the errors.

Reply:
IMO, `CodeTransformConstants.VARIABLE` was more readable (easier to tell the context of the constant) -- but after looking around, this doesn't seem to be natural JS where a class is expected to be instanced and _used_ for something, so I can change it to export the variables/functions directly.